Content.Server/GameObjects/Components/Movement/ClimbableComponent.cs

```diff
@@ -185,11 +185,11 @@ private async void TryMoveEntity(IEntity user, IEntity entityToMove)
                 // we may potentially need additional logic since we're forcing a player onto a climbable
                 // there's also the cases where the user might collide with the person they are forcing onto the climbable that i haven't accounted for
 
-                var othersMessage = Loc.GetString("{0:theName} forces {1:theName} onto {2:theName}!", user,
-                    entityToMove, Owner);
+                var othersMessage = Loc.GetString("{0:theName} forces {1:theName} onto the {2:theName}!", user,
+                    entityToMove, Owner.Name);
```

Comment:
Why is only 1 IEntity using its Name property?

Reply:
I must have missed a piece. My bad. Do I submit a new commit to fix it?

Reply:
Yeah just make a new commit and resolve once it's done. TBH I'm pretty sure I bulldozed it for new physics so it probably won't matter soon.